From 39dab7df89b25ebec59117ff5c16e3e402da2a83 Mon Sep 17 00:00:00 2001
From: Thomas Goirand <zigo@debian.org>
Date: Thu, 8 Oct 2015 11:13:23 -0700
Subject: Removes sphinx.ext.intersphinx to avoid network access

 Upstream is using intersphinx when building the sphinx doc. This patch removes
 sphinx.ext.intersphinx from docs/conf.py to avoid any network access during
 the build of babel.
Forwarded: no
Last-Update: 2014-02-19

Patch-Name: fix-sphinx-conf.py-to-avoid-intersphinx.patch
---
 docs/conf.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 33d9a67..a13d8cc 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -27,7 +27,6 @@ sys.path.append(os.path.abspath('_themes'))
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.extlinks']
 
 # Add any paths that contain templates here, relative to this directory.
